### PR TITLE
Add translations for de, en, it, fr #60

### PR DESCRIPTION
**Pull request by elektrolytmangel**
Add i18n to the project and translate to fr, it, en additional to de. #60

### CODE REVIEW

File: app/src/components/FieldInfo.jsx
Comment (elektrolytmangel) [inline app/src/components/FieldInfo.jsx:1]: translate text before handing it over with props

Comment (elektrolytmangel) [reply app/src/components/FieldInfo.jsx:1]: not a good idea. think it new when needed :)

File: app/src/components/SelectForm.jsx [hunk 1]
```diff
@@ -56,7 +69,7 @@ const SelectForm = ({ labelText, placeholder, name, control, rules, errors, erro
             );
           }}
         />
-        {errors && errors[name] && <Text style={styles.errorTextStyle}>{errorMsg ?? errors[name]?.message}</Text>}
+        {errors && errors[name] && <Text style={styles.errorTextStyle}>{t(errorMsg) ?? errors[name]?.message}</Text>}
```
Comment (elektrolytmangel) [inline app/src/components/SelectForm.jsx:72]: fix this line

File: app/src/components/SwitchForm.js [hunk 2]
```diff
@@ -1,26 +1,20 @@
...
-            onBlur={field.onBlur}
-            onValueChange={field.onChange}
-            value={field.value}
-            disabled={disabled}
-          />
-        )}
+        render={({ field }) => <Switch onBlur={field.onBlur} onValueChange={field.onChange} value={field.value} disabled={disabled} />}
       />
-      {(errors && errors[name]) && <Text style={styles.errorTextStyle}>{errorMsg}</Text>}
+      {errors && errors[name] && <Text style={styles.errorTextStyle}>{errorMsg}</Text>}
```
Comment (elektrolytmangel) [inline app/src/components/SwitchForm.js:17]: translate this message

File: app/src/i18n/i18n.js
Comment (chnuessli) [inline app/src/i18n/i18n.js:1]: A person who is good in translations have approved this. But, neither in IT or FR is she fluent. But she think it's okay :)

**Review by chnuessli — APPROVED** [review]

**Pull request merged**